code/modules/materials/materials.dm

```diff
@@ -0,0 +1,170 @@
+/datum/material
+	var/name = "material"
+	var/color = "#FFFFFF"
+	var/force_multiplier = 1
+	var/health_multiplier = 1
+	var/prefix = "material"
+	var/starting_material = 0
+	var/atom/my_atom
+	var/can_process = 0
+	var/conduction = 0
+
+/datum/material/proc/fire_act()
+	return
+
+/datum/material/proc/throw_impact(atom/hit_atom)
+	return
+
+/datum/material/proc/init_material(var/atom/A)
+	return
+
+/datum/material/iron
+	name = "Iron"
+	color = "#908D8D"
+	force_multiplier = 1
+	health_multiplier = 1
+	prefix = "iron"
+	conduction = 1
+
+/datum/material/gold
+	name = "Gold"
+	color = "" // Handled by the material init. proc.
+	force_multiplier = 0.25
+	health_multiplier = 0.25 // this aint digmineconstructcraft buddy
+	prefix = "gold"
+	conduction = 1
+
+
+/datum/material/gold/init_material(var/atom/A) // Borrowed from GUN_HOG's old EUK code
+	if(istype(A, /obj/item))
+		var/obj/item/I = A
+		I.unacidable = 1
+	var/icon/I
+	var/icon/P = new /icon
+	for(var/iconstate in icon_states(A.icon))
+		var/icon/O = new('icons/effects/material_overlays.dmi', "gold") //Oooh, shiny!
+		I = new(A.icon, iconstate)
+		O.Blend(I, ICON_ADD) //Trim the shine to the item only and add some vibrance to the item.
+		P.Insert(O, iconstate) //Build a new icon set to use with the item itself.
+
+	A.icon = P //Apply the new icon.
+	return
+
+/datum/material/diamond
+	name = "Diamond"
+	color = "#BFFFFF"
+	force_multiplier = 2
+	health_multiplier = 4
```

Comment:
Compared to glass diamonds aren't fragile, but they're definitely not unbreakable wizard gems; they can and do break if hit right, even if you're hitting them with something less hard.